offline.appcache

```diff
@@ -0,0 +1,44 @@
+CACHE MANIFEST
+# 2016-01-14 v1.1.3
```

Comment:
AppCache only checks if there are new files if something changed in the manifest. Normally you'd have an automated tool generate the manifest and add a hash to the top of the file, but since we do it manually I added a version number.

It's an arbitrarily chosen number based on how many times I tried with more and more files.

See http://alistapart.com/article/application-cache-is-a-douchebag